src/citrine/resources/dataset.py

```diff
@@ -3,6 +3,18 @@
 from typing import TypeVar, List, Optional, Iterable, Union, Tuple
 from uuid import UUID
 
+from gemd.entity.base_entity import BaseEntity
+from gemd.entity.link_by_uid import LinkByUID
+from gemd.entity.object import MeasurementSpec, MeasurementRun, MaterialSpec, MaterialRun, \
+    ProcessSpec, ProcessRun, IngredientSpec, IngredientRun
+from gemd.entity.template.condition_template import ConditionTemplate
+from gemd.entity.template.material_template import MaterialTemplate
+from gemd.entity.template.measurement_template import MeasurementTemplate
+from gemd.entity.template.parameter_template import ParameterTemplate
+from gemd.entity.template.process_template import ProcessTemplate
+from gemd.entity.template.property_template import PropertyTemplate
```

Comment:
```suggestion
from gemd.entity.template import PropertyTemplate, MaterialTemplate, MeasurementTemplate, \
    ParameterTemplate, ProcessTemplate, ConditionTemplate
```

Reply:
The former is what my IDE auto-cleans up the imports as. I agree with the suggestion, but I don't think it's worth the effort to spend time being concerned about how imports are formatted.